test/cases.py

```diff
@@ -38,28 +39,28 @@ def forward(self, data: MutableMapping):
         return self.net(x)
 
 
-# Add test cases here
-CASES_NO_DEVICE = [
+# Add test cases for numerically unstable inverses here
+INV_CASES_NO_DEVICE = [
     ###############################################################################
     #                                CLASSIFICATION                               #
     ###############################################################################
     # softmax cross-entropy loss
     {
-        "model_func": lambda: Sequential(Linear(10, 5), ReLU(), Linear(5, 2)),
+        "model_func": lambda: Sequential(Linear(10, 5), ReLU(), Linear(5, 3)),
         "loss_func": lambda: CrossEntropyLoss(reduction="mean"),
         "data": lambda: [
-            (rand(3, 10), classification_targets((3,), 2)),
-            (rand(4, 10), classification_targets((4,), 2)),
+            (rand(3, 10), classification_targets((3,), 3)),
+            (rand(4, 10), classification_targets((4,), 3)),
         ],
         "seed": 0,
     },
     # same as above, but uses reduction='sum'
     {
-        "model_func": lambda: Sequential(Linear(10, 5), ReLU(), Linear(5, 2)),
+        "model_func": lambda: Sequential(Linear(10, 5), ReLU(), Linear(5, 3)),
         "loss_func": lambda: CrossEntropyLoss(reduction="sum"),
         "data": lambda: [
-            (rand(3, 10), classification_targets((3,), 2)),
-            (rand(4, 10), classification_targets((4,), 2)),
+            (rand(3, 10), classification_targets((3,), 3)),
+            (rand(4, 10), classification_targets((4,), 3)),
```

Comment:
Any reason why these cases changed?

Reply:
Not really, I think I simply preferred having a test case with more than 2 classes, but didn't have a particular failure mode in mind.